packages/client/tests/functional/extensions/enabled/itx.ts

```diff
@@ -0,0 +1,212 @@
+import { faker } from '@faker-js/faker'
+import { expectTypeOf } from 'expect-type'
+
+import testMatrix from './_matrix'
+// @ts-ignore
+import type { PrismaClient } from './node_modules/@prisma/client'
+
+declare let prisma: PrismaClient
+
+const email = faker.internet.email()
+
+testMatrix.setupTestSuite(() => {
```

Comment:
This PR is mostly a type change so I thought it would be nice to also test itx at runtime via extensions for some basic sanity checks.